tests/unit/test_package.py

```diff
@@ -76,212 +83,196 @@ def test_sam_injects_policy(sample_app,
     assert 'Role' not in template['Resources']['APIHandler']['Properties']
 
 
-def test_sam_injects_swagger_doc(sample_app,
-                                 mock_swagger_generator,
-                                 mock_policy_generator):
-    p = package.SAMTemplateGenerator(mock_swagger_generator,
-                                     mock_policy_generator)
-    mock_swagger_generator.generate_swagger.return_value = {
-        'swagger': 'document'
-    }
+def test_sam_injects_swagger_doc(sample_app, cfn_gen):
     config = Config.create(chalice_app=sample_app,
                            api_gateway_stage='dev')
-    template = p.generate_sam_template(config)
+    template = cfn_gen.generate_sam_template(config)
     properties = template['Resources']['RestAPI']['Properties']
     assert properties['DefinitionBody'] == {'swagger': 'document'}
 
 
-def test_can_inject_environment_vars(sample_app,
-                                     mock_swagger_generator,
-                                     mock_policy_generator):
-    p = package.SAMTemplateGenerator(
-        mock_swagger_generator, mock_policy_generator)
-    mock_swagger_generator.generate_swagger.return_value = {
-        'swagger': 'document'
-    }
+def test_can_inject_environment_vars(sample_app, cfn_gen):
     config = Config.create(
         chalice_app=sample_app,
         api_gateway_stage='dev',
         environment_variables={
             'FOO': 'BAR'
         }
     )
-    template = p.generate_sam_template(config)
+    template = cfn_gen.generate_sam_template(config)
     properties = template['Resources']['APIHandler']['Properties']
     assert 'Environment' in properties
     assert properties['Environment']['Variables'] == {'FOO': 'BAR'}
 
 
-def test_chalice_tag_added_to_function(sample_app,
-                                       mock_swagger_generator,
-                                       mock_policy_generator):
-    p = package.SAMTemplateGenerator(
-        mock_swagger_generator, mock_policy_generator)
-    mock_swagger_generator.generate_swagger.return_value = {
-        'swagger': 'document'
+def test_can_extract_env_vars_as_cfn_params(sample_app, cfn_gen):
+    config = Config.create(
+        chalice_app=sample_app,
+        api_gateway_stage='dev',
+        environment_variables={
+            'MY_ENV_VAR': 'BAR'
```

Comment:
Could we also get a test to make sure it capitalizes properly from lowercase? So `my_env_var` becomes `MyEnvVar`